### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by renatovieiradesouza**
No need sudo command for this installation.

I am remove sudo command of the line 89.

Now the installation is successful.

### CODE REVIEW

**Comment by wiliansilvazup** [comment]
hello @renatovieiradesouza 

in somes CI/CD pipeline is necessary because we can't move binary to location `/usr/local/bin`, them we can aprove if you create custom option for the script:
Run sudo required by default, but if passed an new argument how by example `DISABLE_SUDO` the execution for mv with condition, see below:
```
DISABLE_SUDO=$2
...
if [ "$DISABLE_SUDO" == "true" ]; then
    mv ./horusec "$INSTALL_PATH"
else
    sudo mv ./horusec "$INSTALL_PATH"
fi
```

**Comment by renatovieiradesouza** [comment]
> hello @renatovieiradesouza
> 
> in somes CI/CD pipeline is necessary because we can't move binary to location `/usr/local/bin`, them we can aprove if you create custom option for the script:
> Run sudo required by default, but if passed an new argument how by example `DISABLE_SUDO` the execution for mv with condition, see below:
> 
> ```
> DISABLE_SUDO=$2
> ...
> if [ "$DISABLE_SUDO" == "true" ]; then
>     mv ./horusec "$INSTALL_PATH"
> else
>     sudo mv ./horusec "$INSTALL_PATH"
> fi
> ```

Hum. I installed with success without sudo. Really need? In this case i can run script with command sudo, example, sudo ./script.sh.

Anyway, this a good solution.

Thank you Willian

**Review by nathanmartinszup — APPROVED** [review]

File: deployments/scripts/install.sh [hunk 1]
```diff
@@ -86,7 +86,7 @@ horusecDownloadAndInstall () {
 
     chmod +x ./horusec
 
-    sudo mv ./horusec "$INSTALL_PATH"
+    mv ./horusec "$INSTALL_PATH"
```
[the remark this answers is not in the document]
Comment (wiliansilvazup) [reply deployments/scripts/install.sh:89]: And add code `$DISABLE_SUDO=$2` after `VERSION_DOWNLOAD=$1`

**Comment by wiliansilvazup** [comment]
i'll close this PR for now, any update we can re open it :)

**Pull request closed without merging**